rest/src/main/kotlin/com/gitlab/kordlib/rest/builder/message/EmbedBuilder.kt

```diff
@@ -92,6 +96,23 @@ class EmbedBuilder : RequestBuilder<EmbedRequest> {
         fields += Field().apply(builder)
     }
 
+    /**
+     * Adds a new [Field] using the given [name] and [value].
+     *
+     * [inline] defines wither a field should be inlined with the default  value set to false.
```

Comment:
`wither` -> `whether`, extra space there near the end as well.

This should probably use the `@param` syntax instead, it's kind of in that format already.